app/src/main/java/org/permanent/permanent/network/NetworkClient.kt

```diff
@@ -50,8 +50,8 @@ import java.util.Date
 
 
 class NetworkClient(private var okHttpClient: OkHttpClient?, context: Context) {
-    private val baseUrl: String = BuildConfig.BASE_API_URL
     private val retrofit: Retrofit
+    private val retrofitNewBaseUrl: Retrofit
```

Comment:
It would have been good to add the base url using dependency injection. The fact that we can't due to the fact that Network client is a singleton, means that we need to duplicate the base URL variable and also have two  retrofit variables.